rubygem-foreman_openscap/rubygem-foreman_openscap.spec

```diff
@@ -14,12 +14,12 @@
 %global foreman_bundlerd_dir %{foreman_dir}/bundler.d
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.5.2
+Version: 0.5.3
 Release: 2%{?foremandist}%{?dist}
 Summary: Foreman plug-in for displaying OpenSCAP audit reports
 Group: Applications/System
 License: GPLv3
-URL: https://github.com/OpenSCAP/foreman_openscap
+URL: https://github.com/theforeman/foreman_openscap
```

Comment:
This may be correct, but doesn't match the gemspec - please update it there too.